core/common/src/androidTest/kotlin/com/mariqzw/supportorganizationsapp/common/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.mariqzw.supportorganizationsapp.common
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("com.mariqzw.common.test", appContext.packageName)
```

Comment:
Тест не валиден, неправильное имя пакета

```suggestion
        assertEquals("com.mariqzw.supportorganizationsapp.common.test", appContext.packageName)
```